Framework/HistogramData/test/HistogramTest.h

```diff
@@ -1119,8 +1128,9 @@ class HistogramTestPerformance : public CxxTest::TestSuite {
 
   HistogramTestPerformance() : xData(histSize, LinearGenerator(0, 2)) {
     BinEdges edges(histSize, LinearGenerator(0, 2));
+    Counts counts(histSize - 1, LinearGenerator(0, 2));
     for (size_t i = 0; i < nHists; i++)
-      hists.push_back(Histogram(edges));
+      hists.push_back(Histogram(edges, counts));
```

Comment:
Why do you need the data in a performance test? `Histogram` initializes it to zero and that should be fine, I think.